README.md

```diff
@@ -92,7 +104,17 @@ The Admin sign in through sign-in page and he will be redirected to the dashboar
 
 ## The MVP :rainbow:
 
- GIF Image
+ - Web
+ 
+![](https://i.imgur.com/KtYlkTi.gif)
+
+![](https://i.imgur.com/CaNT7yA.gif)
+
+
+
+- Mobile
+![](https://i.imgur.com/0MXdmo2.gif)
+
  
 ## Application Setup :mag: 
```

Comment:
In Application Setup you should write more about Mongo configurations, what should the person do to run database, and where to go. 
Just simple and short steps: 

starting with [create account ](https://account.mongodb.com/account/login?n=%2Fv2%2F5ddcde7c79358e92652b7c3a&nextHash=%23metrics%2FreplicaSet%2F5ddce29bbc1dc0c78af6810e%2Fexplorer%2Ftest%2Fbooks%2Ffind) for example